sp800-63-3/definitions.md

```diff
@@ -61,6 +61,9 @@ Verifying the identity of a user, process, or device, often as a prerequisite to
 #### Authentication Factor
 The three types of authentication factors are *something you know*, *something you have*, and *something you are*. Every authenticator has one or more authentication factors.
 
+#### Authentication Intent
+The process of confirming the intent of the claimant to authenticate or reauthenticate by including a manual process in the authentication flow. Some authenticators (e.g., OTP devices) establish authentication intent as part of their operation, others require a specific step, such as pressing a button, to establish intent. Authentication intent is a countermeasure against use by malware of the endpoint as a proxy for authenticating an attacker without the subscriber's knowledge.
```

Comment:
a nit, but manual stands out to me as not accurate.  Intent could be a spoken phrase. External process? Just process?